lib/ruby-mws/api/feed.rb

```diff
@@ -197,7 +197,7 @@ def get_addition_data(xml, entry_hash)
         xml.ProductData {
           xml.Books {
             xml.ProductType {
-              xml.BooksMisc {
+              xml.AbisBook {
```

Comment:
Have you verified that the AbisBook template doesn't have new "required fields" requirements (vs. BooksMisc) ?
I can't search or get to the template specs because I don't have a MWS account.

Reply:
Yes I reviewed their XSD file and:
1) All the fields we add are also applicable to AbisBook.
2) There are no required fields in AbisBook.

Reply:
@tcampbell-blurb this is the xsd schema for book submissions that I found in seller central's help directory https://images-na.ssl-images-amazon.com/images/G/01/rainier/help/xsd/release_1_9/Books.xsd

If you look under the abis books tag, every subelement has minOccurs=0 which I interpret to mean is optional.

The documentation is pretty sparse and hard to follow, I'm hoping that this at least gives us a more useful error message even if it doesn't solve the issues outright.